cider/exec/template/CodeGenerator.h

```diff
@@ -217,6 +217,7 @@ class CodeGenerator {
 
   llvm::Value* codegen(const Analyzer::InIntegerSet* expr, const CompilationOptions& co);
 
+  // Deprecating
```

Comment:
Use TODO instead to call this out. No need to follow previous style.

Reply:
Done.